jetstream_pt/config.py

```diff
@@ -49,6 +49,12 @@
 flags.DEFINE_bool(
     "quantize_kv_cache", None, "defaults to the same value as quantize_weights"
 )
+flags.DEFINE_bool(
+    "internal_quantize_embedding_layer",
```

Comment:
Does internal_* means this is not a public flag? If yes, it doesn't comply with the current naming convention. If not, it's misleading.

Reply:
done.